ghcup-prereleases-0.0.7.yaml

```diff
@@ -540,6 +540,92 @@ ghcupDownloads:
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc-961alpha2-64-deb9
+        A_64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 694d70b648853da04b115d7cdd4fb9bd953fb6361638ee83dda1f7fc8fb344f7
+              dlSubdir: ghc-9.6.0.20230128-x86_64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: 19d8d59a6d68c3b3174a8e3513d3b9236ec8f505428e7e909af4e7b108716416
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-alpine3_12-linux-static-int_native.tar.xz
+          Linux_CentOS:
+            ( >= 7 && < 8 ): &ghc-961alpha2-64-centos7
+              dlHash: 974513010e7d34ccfe01a0b83f31590b0bbc8f1104414ca619e72394b33cc3d0
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-centos7-linux.tar.xz
+            unknown_versioning: *ghc-961alpha2-64-centos7
+          Linux_Debian:
+            (>= 10 && < 11): &ghc-961alpha2-64-deb10
+              dlHash: ac5d97dcdf3d16a2015421cb2561bf800202385ef492e0a6bab529d7dff00d5c
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-deb10-linux.tar.xz
+            < 10: &ghc-961alpha2-64-deb9
+              dlHash: 8bfc3c7d3fd17bfcef6e49adcdf5d51d472e483802c0a7f35dcbc7853402da22
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-deb9-linux.tar.xz
+            '>= 11': &ghc-961alpha2-64-deb11
+              dlHash: 3834de0e46bdeb00bcc4a4b77c1bcccac1b3ea9da856ca713ff267d10bb9e90f
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-deb11-linux.tar.xz
+            unknown_versioning: *ghc-961alpha2-64-deb11
+          Linux_Fedora:
+            '>= 33': &ghc-961alpha2-64-fedora33
+              dlHash: 0bb20e3737a5447623ce8d9d9b5f4b4e71d969cbd0d234c264441dc45bd78bd2
+              dlSubdir: ghc-9.6.0.20230128-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc-961alpha2-64-centos7
+          Linux_Mint:
+            < 20: *ghc-961alpha2-64-deb9
+            '>= 20': *ghc-961alpha2-64-deb10
+          Linux_RedHat:
+            unknown_versioning: *ghc-961alpha2-64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-961alpha2-64-deb9
+            unknown_versioning: *ghc-961alpha2-64-deb10
```

Comment:
Why don't we use the ubuntu bindists?

* https://downloads.haskell.org/ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-ubuntu18_04-linux.tar.xz
* https://downloads.haskell.org/ghc/9.6.0.20230128/ghc-9.6.0.20230128-x86_64-ubuntu20_04-linux.tar.xz

Reply:
Using the Ubuntu 18.04 bindist in the first case would be reasonable. We will certainly try to do so with the next pre-release; however, my vague sense is that it would be better to keep things as-is for this release as the current configuration has already passed [our smoke-test](https://gitlab.haskell.org/ghc/ghcup-ci/-/jobs/1346076).

However, I am honestly not sure what to do about the `unknown_versioning` case; there is by definition no robust solution here.

Reply:
Actually I'm thinking Rocky linux 8 might be ok... Centos 7 is still on ncurses 5, which is decommissioned in newer Fedora versions.